brew.sh

```diff
@@ -95,6 +95,3 @@ brew install ssh-copy-id
 brew install tree
 brew install vbindiff
 brew install zopfli
-
-# Remove outdated versions from the cellar.
-brew cleanup
```

Comment:
Why remove this?

Reply:
It’s no longer necessary! Brew does this automatically now on a timeline (as of [Brew v2.0.0](https://brew.sh/2019/02/02/homebrew-2.0.0/)), whenever a `brew update` is performed.

From the change notes:

> `brew cleanup` is run periodically (every 30 days) and triggers for individual formula cleanup on reinstall, install or upgrade. You can opt-out of this behaviour by setting the `HOMEBREW_NO_INSTALL_CLEANUP` variable. This addresses a long-standing complaint where users were surprised by how much disk space Homebrew used if they did not run `brew cleanup`.

Reply:
I went ahead & added this back in, since you had a question about it, and it’s harmless to keep.